src/gui/generalsettings.cpp

```diff
@@ -138,6 +159,29 @@ void GeneralSettings::loadMiscSettings()
     _ui->newFolderLimitSpinBox->setValue(newFolderLimit.second);
     _ui->newExternalStorage->setChecked(cfgFile.confirmExternalStorage());
     _ui->monoIconsCheckBox->setChecked(cfgFile.monoIcons());
+
+    // we assume the language names have been loaded into the dropdown already (by making sure that the corresponding method is called before this one)
+    const auto &locale = cfgFile.uiLanguage();
```

Comment:
I guess this could also be moved to loadLanguageNamesIntoDropdown?
By now we have 3 separate code parts which in the end all do one thing.

as the loading always happens at the beginning I don't expect any issues by populating the combobox in loadMiscSettings

Reply:
So you're suggesting to either call the utility method that populates the dropdown in `loadMiscSettings` or even integrate it here?